README-fr.adoc

```diff
@@ -0,0 +1,310 @@
+
+include::_settings-README.adoc[]
+
+{uri-project}/[Asciidoctor] est un processeur de texte _rapide_ et une chaîne de publication pour convertir du contenu {uri-what-is-asciidoc}[AsciiDoc] en HTML5, DocBook 5 (ou 4.5) et d'autres formats.
+Asciidoctor est écrit en Ruby, packagé sous forme de RubyGem et publié sur {uri-rubygem}[RubyGems.org].
+La gem est aussi incluse dans plusieurs distributions Linux, incluant Fedora, Debian et Ubuntu.
+Asciidoctor est open source, {uri-repo}[hébergé sur GitHub] et distribué sous {uri-license}[la licence MIT].
+
+.Documentation clée
+[.compact]
+* {uri-docs}/what-is-asciidoc[Qu'est ce qu'Asciidoc ?]
+* {uri-docs}/asciidoc-writers-guide[Guide pour Ecrivain AsciiDoc]
+* {uri-docs}/asciidoc-syntax-quick-reference[Syntaxe de Référence Asciidoc]
+* {uri-docs}/user-manual[Manuel Utilisateur Asciidoctor]
+
+.Là où Ruby va, Asciidoctor le suit
+****
+Vous pouvez exécuter Asciidoctor dans le JVM en utilisant JRuby.
+Pour invoquer l'API Asciidoctor directement de Java ou d'autres langages de JVM, utilisez {uri-asciidoctorj}[AsciidoctorJ].
+Il y a des plugins disponibles, basés sur AsciidoctorJ, qui intègrent le processeur Asciidoctor dans des builds Apache Maven, Gradle ou Javadoc.
+
+Asciidoctor s'exécute également au sein de JavaScript.
+Nous utilisons http://opalrb.org[Opal] pour transcrire le code source Ruby en JavaScript afin de produire {uri-asciidoctorjs}[Asciidoctor.js], une version pleinement fonctionnelle d'Asciidoctor qui s'intègre dans tout environnement JavaScript, comme un navigateur web ou Node.js.
+Asciidoctor.js est utilisé pour faire fonctionner les extensions AsciiDoc Preview pour Chrome, Atom, Brackets et autres outils web.
+****
+
+ifdef::badges[]
+.*Santé du projet*
+image:https://img.shields.io/travis/asciidoctor/asciidoctor/master.svg[Build Status (Travis CI), link=https://travis-ci.org/asciidoctor/asciidoctor]
+image:https://ci.appveyor.com/api/projects/status/ifplu67oxvgn6ceq/branch/master?svg=true&amp;passingText=green%20bar&amp;failingText=%23fail&amp;pendingText=building%2E%2E%2E[Build Status (AppVeyor), link=https://ci.appveyor.com/project/asciidoctor/asciidoctor]
+//image:https://img.shields.io/coveralls/asciidoctor/asciidoctor/master.svg[Coverage Status, link=https://coveralls.io/r/asciidoctor/asciidoctor]
+image:https://codeclimate.com/github/asciidoctor/asciidoctor/badges/gpa.svg[Code Climate, link="https://codeclimate.com/github/asciidoctor/asciidoctor"]
+image:https://inch-ci.org/github/asciidoctor/asciidoctor.svg?branch=master[Inline docs, link="https://inch-ci.org/github/asciidoctor/asciidoctor"]
+endif::[]
+
+== La vue d'ensemble
+
+Asciidoctor lit du contenu écrit en texte simple, comme présenté dans la partie gauche de l'image ci-dessous, et le convertit en HTML5, comme présenté dans la partie droite.
+Asciidoctor applique une feuille de style par défaut au document HTML5 afin de fournir une expérience de lecture clé en main et agréable.
```

Comment:
Maybe we could invert "agréable" and remove "et": "une expérience de lecture *agréable*, clé en main"